src/Tools/dotnet-monitor/Program.cs

```diff
@@ -35,6 +35,22 @@ class Program
                 Urls(), MetricUrls(), ProvideMetrics(), DiagnosticPort(), NoAuth()
               };
 
+        private static Command ConfigCommand() =>
+              new Command(
+                  name: "config",
+                  description: "Configuration related commands for dotnet-monitor.")
+              {
+                new Command(
+                  name: "show",
+                  description: "Shows configuration, as if dotnet-monitor collect was executed with these parameters.")
+                {
+                        // Handler
+                        CommandHandler.Create(Delegate.CreateDelegate(typeof(Func<CancellationToken, IConsole, string[], string[], bool, string, bool, ConfigDisplayLevel, Task<int>>),
+                            new DiagnosticsMonitorCommandHandler(), nameof(DiagnosticsMonitorCommandHandler.ShowConfig))),
+                        Urls(), MetricUrls(), ProvideMetrics(), DiagnosticPort(), NoAuth(), ConfigLevel()
```

Comment:
I have a feeling we're going to forget to add new options here when adding them to the `collect` command. Is there a way you can refactor this so that they are shared among both commands? I know this is using initializer syntax, which might prevent the sharing, but I think a refactor could save us in the future.

Feel free to not do this now. We can refactor stuff later in the next preview.